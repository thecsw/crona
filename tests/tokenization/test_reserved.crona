int
bool
byte
string
void
if
else
while
return
array
true
false
read
write
havoc