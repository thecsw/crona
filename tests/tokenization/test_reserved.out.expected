INT [1,1]
BOOL [2,1]
BYTE [3,1]
STRING [4,1]
VOID [5,1]
IF [6,1]
ELSE [7,1]
WHILE [8,1]
RETURN [9,1]
ARRAY [10,1]
TRUE [11,1]
FALSE [12,1]
READ [13,1]
WRITE [14,1]
HAVOC [15,1]
EOF [15,6]
